src/components/merge_results/fields/CompareContributor.tsx

```diff
@@ -0,0 +1,16 @@
+import { Contributor } from '../../../types/contributor.types';
+import { ContributorBox } from './ContirbutorBox';
+
+interface CompareContributorProps {
+  sourceContributor?: Contributor;
+  targetContributor?: Contributor;
+}
+
+export const CompareContributor = ({ sourceContributor, targetContributor }: CompareContributorProps) => {
```

Comment:
Denne komponenten gir ikke så mye verdi nå, og innholdet kunne like godt vært flytta ett hakk opp.
Har foreløpig gjort det slik for å følge strukturen til andre felter, og det vil etter stor sannsynlighet bli behov for det uansett når man skal vise mer info om sammenligning av bidragsytere i fremtiden